.github/workflows/reusable_benchmarks.yml

```diff
@@ -135,22 +140,21 @@ jobs:
 
     - name: Run UMF benchmarks
       id: benchmarks
-      working-directory: ${{env.BUILD_DIR}}
       run: >
-        source ${{github.workspace}}/.venv/bin/activate &&
-        taskset -c ${{ env.CORES }} ${{ github.workspace }}/sycl-repo/unified-runtime/scripts/benchmarks/main.py
+        source .venv/bin/activate &&
+        taskset -c ${{ env.CORES }} ./sc/devops/scripts/benchmarks/main.py
         ~/bench_workdir_umf
         --umf ${{env.BUILD_DIR}}
-        --compare baseline
         --timeout 3000
-        ${{ inputs.upload_report && '--output-html' || '' }}
-        ${{ inputs.pr_no != 0 && '--output-markdown' || '' }}
+        --output-html remote
+        --results-dir ${{ github.workspace }}/results-repo
+        --output-markdown
         ${{ inputs.bench_script_params }}
 
     # In case it failed to add a comment, we can still print the results.
     - name: Print benchmark results
-      if: ${{ always() && inputs.pr_no != 0 }}
-      run: cat ${{env.BUILD_DIR}}/benchmark_results.md
+      if: ${{ always() }}
+      run: cat ${{ github.workspace }}/benchmark_results.md || true
```

Comment:
Isn't it in the `${{ github.workspace }}/results-repo/`?

Reply:
nope, we commit and push results to the branch (checked-out in `results-repo`), but the md file is generated locally

// you can see the path to generated file in the benchmark logs: https://github.com/oneapi-src/unified-memory-framework/actions/runs/13971793856/job/39115241536#step:12:136